packages/main/src/SegmentedButtonItem.ts

```diff
@@ -105,6 +105,13 @@ class SegmentedButtonItem extends UI5Element implements IButton, ISegmentedButto
 	@property()
 	icon!: string;
 
+	/**
+	 * Defines if a content has been added to the default slot
```

Comment:
please add more descriptive description

Reply:
Is it better now? It was just copied from the button, as it is a private property.